### PR TITLE
Update livebook

### PR DESCRIPTION
**Pull request by msluszniak**

### CODE REVIEW

**Review by msluszniak — COMMENTED** [review]
And I think after new release of Nx it will be worth to change also the line with declaration of pi to use `Nx.Constant.pi`

File: nx/guides/intro-to-nx.livemd [hunk 1]
```diff
@@ -264,7 +264,7 @@ suggest that they would work on primitive values, called scalars.
 Indeed, a tensor can be a scalar:
 
 ```elixir
-pi = Nx.tensor(3.1415, type: :f32)
+pi = Nx.Constants.pi()
```
Comment (josevalim) [inline nx/guides/intro-to-nx.livemd:267]: I think in this case we want to show how to build it by hand, so we should likely revert. :)

Comment (josevalim) [reply nx/guides/intro-to-nx.livemd:267]:
```suggestion
pi = Nx.tensor(3.1415, type: :f32)
```

File: nx/guides/intro-to-nx.livemd [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ```elixir
 Mix.install([
-  {:nx, "~> 0.2"}
+  {:nx, github: "elixir-nx/nx", override: true, sparse: "nx"}
```
Comment (josevalim) [inline nx/guides/intro-to-nx.livemd:5]:
```suggestion
  {:nx, "~> 0.5"}
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**